NuRadioMC/simulation/simulation.py

```diff
@@ -1311,7 +1311,6 @@ def _create_station_output_structure(self, n_showers, n_antennas):
     def _read_input_particle_properties(self, idx=None):
         if idx is None:
             idx = self._primary_index
-        self._fin['n_interaction'][self._shower_index] = self._fin['n_interaction'][idx]
```

Comment:
can you also change the if in l 1328 from self._fin['n_interaction'][self._shower_index] to self._fin['n_interaction'][idx] ...

Reply:
good point, done